src/Jackett.Common/Definitions/arabtorrents.yml

```diff
@@ -0,0 +1,220 @@
+    - {id: 16, cat: Audio, desc: "Music"}
+    - {id: 177, cat: Audio, desc: "Flac/Music"}
+    - {id: 127, cat: Audio, desc: "Full-Packs/Music"}
+    - {id: 17, cat: Audio/MP3, desc: "MP3/Music"}
+    - {id: 19, cat: Audio/Video, desc: "Videos/Music"}
+    # OSN
+    - {id: 146, cat: TV/Other, desc: "OSN"}
+    - {id: 147, cat: TV/Other, desc: "Arabic/OSN"}
+    - {id: 153, cat: TV/Other, desc: "English/OSN"}
+    - {id: 179, cat: TV/Other, desc: "Old/OSN"}
+    # Ramadan 2022
+    - {id: 192, cat: Other, desc: "Ramadan 2022"}
+    - {id: 198, cat: Other, desc: "Full Pack/Ramadan 2022"}
+    - {id: 193, cat: Other, desc: "Ramadan HD/Ramadan 2022"}
+    - {id: 195, cat: Other, desc: "Ramadan Khaliji/Ramadan 2022"}
+    - {id: 194, cat: Other, desc: "Ramadan SD/Ramadan 2022"}
+    - {id: 196, cat: Other, desc: "Ramadan Web/Ramadan 2022"}
+    # Sports
+    - {id: 120, cat: TV/Sport, desc: "Sports"}
+    - {id: 159, cat: TV/Sport, desc: "Full Matches/Sports"}
+    - {id: 155, cat: TV/Sport, desc: "Goals/Sports"}
+    # Rest
+    - {id: 102, cat: TV/Other, desc: "Other/SD TV"}
+    - {id: 202, cat: Other, desc: "AT"}
+    - {id: 203, cat: Other, desc: "AT/AT"}
+    - {id: 34, cat: Other, desc: "Misc"}
+    - {id: 132, cat: PC/Mobile-Android, desc: "Android/Misc"}
+    - {id: 163, cat: TV/Documentary, desc: "Documentary/Misc"}
+    - {id: 122, cat: Other, desc: "Other/Misc"}
+    - {id: 121, cat: Other, desc: "Theater/Misc"}
+    - {id: 214, cat: Other, desc: "Other/VIP"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep]
+    movie-search: [q]
+    music-search: [q]
+    book-search: [q]
+
+settings:
+  - name: username
+    type: text
+    label: Username
+  - name: password
+    type: password
+    label: Password
+  - name: sort
+    type: select
+    label: Sort requested from site
+    default: added
+    options:
+      added: created
+      seeders: seeders
+      size: size
+      name: title
+  - name: type
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+
+login:
+  path: takelogin.php
+  method: post
+  inputs:
+    username: "{{ .Config.username }}"
+    password: "{{ .Config.password }}"
+  error:
+    - selector: table:has(td.thead:contains("حدث خطأ"))
+  test:
+    path: index.php
+    selector: a[href*="logout.php"]
+
+download:
+  before:
+    path: takethanks.php
+    method: post
+    inputs:
+      torrentid: "{{ .DownloadUri.Query.id }}"
+
+search:
+  paths:
+    - path: browse.php
+
+  inputs:
+    do: search
+    keywords: "{{ .Keywords }}"
+    # t_name, t_description, t_both, t_uploader, t_genre
+    search_type: t_both
+    # does not support multi category searching so defaulting to all.
+    category: 0
+    include_dead_torrents: yes
+    sort: "{{ .Config.sort }}"
+    order: "{{ .Config.type }}"
+
+  keywordsfilters:
+    - name: re_replace
+      args: ["\\W*\\b\\w{1,3}\\b", " "] # The minimum search term length is 4 characters.
```

Comment:
The minimum length can sometimes be reduced by 1 character by preceding all words with a wildcard such as `%`. This would mean that something like `ant-man and the wasp` wouldn't just become `wasp`.

Can the user test that on the tracker?